src/pages/AssetMapPage/AMDrawer.tsx

```diff
@@ -6,9 +6,8 @@ import { makeStyles, Button } from '@material-ui/core';
 import DnsIcon from '@material-ui/icons/Dns';
```

Comment:
to @crabsinger's point, this sort of unrelated refactoring might be better done in a separate PR, to reduce the amount of regression risk per deployment. in the future, you could create a branch for this first, then create the main branch for the feature off of that one.

you can even do this after the fact, as long as you have a commit with these changes that comes first. you would checkout that commit, create a new branch off of that commit, push the branch, then go back to your latest commit.

Reply:
This isn't unrelated refactoring, this is collecting usages into a component

If you want me to code more responsibly, I will need less aggressive deadlines

Reply:
> This isn't unrelated refactoring, this is collecting usages into a component

ah yes, you're right, sorry, i was going a bit too quickly